cyfronet-fid/marketplace
# [#2132] Don't leak sensitive OMS fields in Offering API

As we used the same OMS serializer both in Offering and Ordering API,
the full OMS response was also served to the Offering API users.

Leave only: id, name, type and custom_params (but don't expose the
default value, only the mandatory flag).

## How to test

Go to the Offering API swagger and see that when listing resources, for example with `GET /api/v1/resources` in the `available_omses` there are only the aforementioned fields.

On the other hand, in Ordering API all the fields should be rendered as before, i.e. `trigger_url`, `default` and `custom_params[].default` should be returned. For example for the `GET /api/v1/oms` (The token: `o922PK12ozhnbbEEsz4t`.)

Fixes: #2132.
